src/orders/components/OrderUnfulfilledProductsCard/OrderUnfulfilledProductsCard.tsx

```diff
@@ -41,6 +41,26 @@ const useStyles = makeStyles(
       "&:hover": {
         backgroundColor: theme.palette.saleor.active[5],
         color: theme.palette.saleor.active[1]
+      },
+      "& > div": {
+        minWidth: 0,
+        whiteSpace: "nowrap",
+        overflow: "hidden",
+        textOverflow: "ellipsis"
+      }
```

Comment:
Great stuff - I think we should have this in every `CardTitle` really, as every user generated input can be really long, not only warehouse names. Could we add this to `CardTitle` by default? Or maybe make it a reusable component that applies those 3 styles (`whiteSpace`, `overflow`, `textOverflow`)?

Reply:
It's not that simple. `textOverflow: ellipsis` is absurdly difficult to work with in responsive design, there has to be specific conditions met for it to work. For example in flexboxes, multiple wrapper elements without specified width/height must be able to shrink below its content size (aparat from `min-width: 0`, `flex-shrink` is necessary). I'll be making a PR real soon handling long strings in all dashboard tables, I'll look into card headers too there.